### DISCUSSION TITLE
Is it possible to enable the clock on just one of my monitors?

### DISCUSSION THREAD

**Discussion by aydenking** [martinet101/ElevenClock#382] [Q&A] [answered]
Hello,

I have a three monitors and a tv connected to my pc, and I'd like to hide the clock on just the tv's taskbar, but still show it on the taskbar of my two secondary monitors. Is this possible currently?

Also to martinet101, thanks for ElevenClock! It's singlehandedly made my Windows 11 experience actually bearable haha.

Thanks!

**Comment by marticliment** [comment, marked as answer]
Not for the moment, I could add some kind of blacklisting... Could you please open a Feature Request?

**Reply by marticliment** [reply]
The issue has been fixed. you can right-click a clock -> monitor tools -> blacklist monitor
